script/hassfest/manifest.py

```diff
@@ -249,6 +249,7 @@ def verify_wildcard(value: str):
         vol.Optional("disabled"): str,
         vol.Optional("iot_class"): vol.In(SUPPORTED_IOT_CLASSES),
         vol.Optional("supported_brands"): vol.Schema({str: str}),
+        vol.Optional("helper"): bool,
```

Comment:
Let's keep it out of this PR, as it also needs updates to websocket and loader.py etc, 
```suggestion
```

Reply:
Removed